analysis/report_templates/default.html

```diff
@@ -168,46 +168,36 @@ <h5 class="center-align">By avg. rank</h5>
 
             <h2 class="green-text text-darken-3">{{ benchmark.name }} summary</h2>
 
-            {% if benchmark.type == 'bug' %}
-
-            <div class="row">
-               <div class="col s12">
-                    <ul class="tabs">
-                      <li class="tab col s6"><a href="#{{ benchmark.name }}_bugs_normal" class="green-text text-darken-3">Bug coverage (linear)</a></li>
-                      <li class="tab col s6"><a href="#{{ benchmark.name }}_bugs_logscale" class="green-text text-darken-3">Bug coverage (log)</a></li>
-                    </ul>
-                </div>
-                <div id="{{ benchmark.name }}_bugs_normal" class="col s12">
-                    <h5 class="center-align">Mean bug coverage growth over time</h5>
-                    <img class="responsive-img materialboxed"
-                         src="{{ benchmark.bug_coverage_growth_plot }}">
-                </div>
-                <div id="{{ benchmark.name }}_bugs_logscale" class="col s12">
-                    <h5 class="center-align">Mean bug coverage growth over time</h5>
-                    <img class="responsive-img materialboxed"
-                         src="{{ benchmark.bug_coverage_growth_plot_logscale }}">
-                </div>
-            </div>
-
-            {% endif %}
-
             <div class="row">
                 <div class="col s6">
+            {% if benchmark.type == 'bug' %}
+                    <h5 class="center-align">Discovered bug coverage distribution</h5>
+                    <img class="responsive-img materialboxed"
+                         src="{{ benchmark.bug_box_plot }}">
+            {% else %}
                     <h5 class="center-align">Ranking by median reached code coverage</h5>
                     <img class="responsive-img materialboxed"
                          src="{{ benchmark.ranking_plot }}">
+            {% endif %}
                 </div>
                 <div class="col s6">
                     <h5 class="center-align">Reached code coverage distribution</h5>
                     <img class="responsive-img materialboxed"
-                         src="{{ benchmark.violin_plot }}">
+                         src="{{ benchmark.box_plot }}">
                 </div>
             </div>
             <div class="row">
                 <div class="col s12">
                     <ul class="tabs">
+            {% if benchmark.type == 'bug' %}
```

Comment:
If it is bug based coverage, put them in the order first, so they should by default first in first tab.

Reply:
bug coverage are the first two tabs now.